_templates/components/explore.html

```diff
@@ -0,0 +1,129 @@
+<h2 class="section-title">Explore <span class="dipy-highlight">DIPY</span></h2>
+<div class="explore pad-t-32 grid-3">
+  <div class="explore-tile pad-h-20 pad-v-48">
+    <div class="explore-tile-icon">
+      <img src="_static/images/icons/cli.png" alt="" srcset="">
+    </div>
+    <h4 class="explore-tile-title mar-v-5">Command-Line Interfaces</h4>
+    <p class="explore-title-description">
+      All the algorithms are available using CLI. You can also create your own algorithms.
+    </p>
+    
+    <a class="btn-link explore-tile-know-more" href="https://dipy.org/documentation/1.7.0/interfaces/">
+      Know more
+    </a>
+    
+  </div>
+  <div class="explore-tile pad-h-20 pad-v-48">
+    <div class="explore-tile-icon">
+      <img src="_static/images/icons/cli.png" alt="" srcset="">
+    </div>
+    <h4 class="explore-tile-title mar-v-5">Command-Line Interfaces</h4>
```

Comment:
In dark mode, the title is not enough emphasized. We do not see the bold effect. Maybe the text titles should be lighter in dark mode (closer to white)

Reply:
Let me work on that